browser/src/map/Clipboard.js

```diff
@@ -977,7 +1005,32 @@ L.Clipboard = L.Class.extend({
 		let clipboardContents;
 
 		try {
-			clipboardContents = await clipboard.read();
+			if (window.ThisIsTheiOSApp) {
+				const encodedClipboardData = await window.webkit.messageHandlers.clipboard.postMessage('read');
```

Comment:
can we put that special iOS handling into separate method? Will be easier to read

Reply:
Sure, that makes sense, let me do that...